### PR TITLE
Use our own Webpack config

### PR DESCRIPTION
**Pull request by ngotchac**
Getting rid of chrome extension scripts in favor of [Chrome Extension Utils](https://github.com/ngotchac/create-chrome-extension). This module exports scripts that could be used in Create Chrome Extension (might want to make a PR // sort this out) that we can use to bundle the Chrome Extension.

Still have `npm start` and `npm run build` scripts to start and watch for modifications (should be auto-updated by chrome if the extensions tab is open) and build and pack the extension.

### CODE REVIEW

File: web3/embed.html [hunk 1]
```diff
@@ -19,12 +19,15 @@
...
+        right: 1em;
         text-align: right;
-        font-size: 1.3em;
+        font-size: 1.2em;
         color: #fff;
-        background: #e66;
-        padding: .5em .5em .2em;
+        background: rgba(40, 40, 40, 0.75);
+        padding: 0 1em;
+        height: calc(1em + 36px);
```
Comment (tomusdrw) [inline web3/embed.html:28]: The same div is used to display information about unsupported version, does it look good with it too? (To test you need to build `beta` branch)

File: web3/styles.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+.iframe__main {
+	position: fixed;
+	bottom: 0;
+	right: 0;
+	background: none;
+	border: none;
+	opacity: 0;
+	animation: .4s ease-out 2s;
+  animation-name: fadeIn;
```
Comment (tomusdrw) [inline web3/styles.css:26]: Some issues with formatting here.

**Review by tomusdrw — COMMENTED** [review]
Looks good! One question awaiting to be answered :)

File: web3/embed.html (hunk 1 above)
Comment (ngotchac) [reply web3/embed.html:28]: Ah ok, I'll try then !

File: web3/styles.css (hunk 2 above)
Comment (ngotchac) [reply web3/styles.css:26]: Ah strange.... Will fix !

File: web3/embed.html (hunk 1 above)
Comment (ngotchac) [reply web3/embed.html:28]: image

File: web3/styles.css (hunk 2 above)
Comment (ngotchac) [reply web3/styles.css:26]: Mix of TABS and SPACES...

**Pull request merged**